test/coverage-report

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+is_patch=$1
+python_binary=$2
+github_commit=$3
+pr_args=$4
+
```

Comment:
Can we add a check that the script requires four arguments? If four arguments are not supplied we fail it